### PR TITLE
Admin api integration

### LINKED ISSUE #131: Ensure auth guard connects to backend
AuthGuard should check if the token exists and not expired.
If the token fails should redirect to login

### LINKED ISSUE #132: Ensure the JWT token is stored correctly
Store the JWT in the correct format and get it using the API

### PR DESCRIPTION
**Pull request by HarryPi**
### Features
- Include a service that returns the endpoints
- Ensures auth guard is functional closes #131 

### Code Changes
- Store the JWT token correctly and get it from backend closes #132
- Display error on failed login #133 .
### Testing

### CODE REVIEW

File: Backend/src/main/java/com/project/omega/controller/UserController.java [hunk 1]
```diff
@@ -68,7 +68,7 @@ public ResponseEntity updateById(@PathVariable(value = "id") Long id,
             userToUpdate.setEmail(userInDb.getEmail());
         }
 
-        if (!updatedUser.getPassword().isEmpty()) {
+        if (updatedUser.getPassword() != null) {
```
Comment (markscamilleri) [inline Backend/src/main/java/com/project/omega/controller/UserController.java:71]: Shouldn't we also check that it's not an empty string?

File: Backend/src/main/resources/application.properties [hunk 2]
```diff
@@ -1,12 +1,4 @@
...
-spring.jpa.database-platform=org.hibernate.dialect.MySQL5InnoDBDialect
-spring.datasource.url=jdbc:mysql://${SECRETS_DB_URL}
-spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
-jwt.secret= ${SECRETS_JWT}
-sendGrid.key= ${SECRETS_SENDGRID}
\ No newline at end of file
+spring.datasource.username = omegaadmin3@omegadatabase3
+spring.datasource.password = omegaDatabase3
+spring.datasource.url=jdbc:mysql://omegadatabase3.mysql.database.azure.com/omegarestapi?useLegacyDatetimeCode=false&serverTimezone=UTC
+jwt.secret= 'a@a.com'
```
Comment (markscamilleri) [inline Backend/src/main/resources/application.properties:4]: This shouldn't have been committed

Comment (HarryPi) [reply Backend/src/main/resources/application.properties:4]: This indeed should have not i excluded it.

File: Backend/src/main/java/com/project/omega/controller/UserController.java (hunk 1 above)
Comment (HarryPi) [reply Backend/src/main/java/com/project/omega/controller/UserController.java:71]: that is true i forgot Java wont check with null. nice catch

File: AdminClient/src/app/toolbar/store/toolbar.reducer.ts [hunk 3]
```diff
@@ -12,7 +12,7 @@ const initialState: State = {
 
 export const selectToolbarState = createFeatureSelector<fromApp.State, State>('toolbar');
 
-export const selectShowProgressBar = createSelector(
+export const selectIsToolbarVisible = createSelector(
```
[the remark this answers is not in the document]
Comment (HarryPi) [reply AdminClient/src/app/toolbar/store/toolbar.reducer.ts:15]: True i just wanted the word visible in there the name change to toolbar was unintentional/

**Pull request merged**